platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java

```diff
@@ -640,6 +643,21 @@ public void setZoom(@FloatRange(from = MapboxConstants.MINIMUM_ZOOM, to = Mapbox
     nativeMapView.setZoom(zoom, focalPoint, 0);
   }
 
+  /**
+   * Moves the center and the zoom of the camera specified by a LatLng object and double zoom.
+   * <p>
+   * Note that at low zoom levels, setLatLng is constrained so that the entire viewport shows map data.
+   * </p>
+   *
+   * @param latLng Target location to change to
+   */
+  public void setLatLngZoom(@NonNull LatLng latLng,
```

Comment:
I don't think we should add this. It's already possible to set center and zoom simultaneously with `moveCamera` -- there's no need to add to what is already a sprawling camera API. If anything, we should be _removing_ individual setters such as `setLatLng` and `setZoom` in favor of the unified `moveCamera`.

Reply:
I initially implemented that api for having a more performant way of doing camera updates with Android SDK animators. After running benchmarks on both setups, the difference is neglectable:

<img width="941" alt="screen shot 2018-02-14 at 07 34 32" src="https://user-images.githubusercontent.com/2151639/36190622-8af04b1c-1159-11e8-93d8-7b34ca5091bb.png">

thank you for flagging @jfirebaugh, I will work on removing the API.

Reply:
@tobrun I'd be great to sync the removal of exposed setters with https://github.com/mapbox/mapbox-plugins-android/issues/259 and https://github.com/mapbox/mapbox-gl-native/issues/11170 as right now exposed setters are the only way to manipulate camera without a loss of tracking mode.

Reply:
happening in https://github.com/mapbox/mapbox-gl-native/pull/11201